File: dsm-platform-lib/src/main/java/repository/ModelTransferSystem.java
package repository;

import lombok.Getter;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@RequiredArgsConstructor
@Setter
public class ModelTransferSystem {
    @NonNull
    private ModelExporter exporter;
    @NonNull
    private ModelImporter importer;
}

File: dsm-platform-lib/src/main/kotlin/ru/psu/repository/ModelTransferSystem.kt
package ru.psu.repository

import ru.psu.repository.transferInterfaces.ModelExporter
import ru.psu.repository.transferInterfaces.ModelImporter

//Класс системы экспорта-импорта моделей
class ModelTransferSystem(
        var exporter: ModelExporter, //Экспортёр моделей
        var importer: ModelImporter //Импортёр моделей
)